### PR TITLE
Add initial actual view and test

### PR DESCRIPTION
**Pull request by Jim3Things**
Also:
-- Add mock for store client, use in actual_test
-- add stretchr\objx as a vendor dependency (needed by testify mock)

### CODE REVIEW

File: simulation/internal/clients/inventory/actual_test.go [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+	"github.com/Jim3Things/CloudChamber/simulation/pkg/errors"
+	pb "github.com/Jim3Things/CloudChamber/simulation/pkg/protos/inventory"
+	"github.com/stretchr/testify/suite"
+)
+
+type standardBladeView struct {
+	ts *actualTestSuite
+}
+
+func (sbv *standardBladeView) set(view *BladeViewActual) {
```
[the remark this answers is not in the document]
Comment (Jim3Things) [reply simulation/internal/clients/inventory/actual_test.go:18]: Yeah, they're not.  I could make it just "(*standardBladeView)".  Thoughts?  Same for the others.

[the remark this answers is not in the document]
Comment (MikeCJohnsonUk) [reply simulation/internal/clients/inventory/actual_test.go:18]: I've no particular preference, but I would anticipate that removing the "sbv" part would make it obvious that is was not required as opposed to accidentally unused.

Your call.

File: simulation/internal/clients/inventory/actual_test.go [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+	require.Equal(pb.Actual_operational, view.GetCondition())
+	require.Equal(pb.BladeState_booting, view.GetState())
+	require.EqualValues(10, view.GetCore().EnteredAt)
+	require.False(view.GetCore().Terminal)
+	require.EqualValues(11, view.GetCore().GetGuard())
+	require.Nil(view.GetTimer())
+}
+
+func (sbv *standardBladeView) saveString() string {
```
[the remark this answers is not in the document]
Comment (MikeCJohnsonUk) [reply simulation/internal/clients/inventory/actual_test.go:39]: As above.

**Review by MikeCJohnsonUk — APPROVED** [review]

File: simulation/internal/clients/inventory/actual_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jim3Things) [reply simulation/internal/clients/inventory/actual_test.go]: I've applied this policy to actual_test.go.  Let's see how it looks.

**Pull request merged**